opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -930,9 +930,25 @@ const CpGridFamily::Traits::LeafIndexSet& CpGrid::leafIndexSet() const
     return *current_view_data_->index_set_;
 }
 
-void CpGrid::globalRefine (int)
-{
-    std::cout << "Warning: Global refinement not implemented, yet." << std::endl;
+void CpGrid::globalRefine (int refCount)
+{
+    assert(refCount>-1);
```

Comment:
This should probably not be an assertion. Either print  a warning or throw an exception.